src/utils/time.ts

```
@@ -1,11 +1,11 @@
-import { TZDate } from '@date-fns/tz';
-import { format } from 'date-fns';
-
-export function getTimeInCity(
-  timezone: string,
-  formatStr = 'yyyy-MM-dd HH:mm:ssXXX',
-): string {
-  const now = new Date();
-  const zonedDate = new TZDate(now, timezone);
-  return format(zonedDate, formatStr);
-}
+import { TZDate } from '@date-fns/tz';
```

ESLint

[error] Unable to resolve path to module '@date-fns/tz'.

(import-x/no-unresolved)